docs/developer-guide/map-query-parameters.md

```diff
@@ -343,7 +343,7 @@ Requirements:
 - The number of layers should match the number of sources
 - The source name can be a string that must match a catalog service name present in the map or an object that defines an external catalog (see example)
 
-Supported layer types are WMS, WMTS, WFS and 3D Tiles.
+Supported layer types are WMS, WMTS, WFS, 3D Tiles and GeoJSON.
 
 Example:
```

Comment:
We could add a section at the bottom to explain how to add a GeoJSON via catalog query param, eg:

---

It is possible to add GeoJSON layer using the following configuration:

```json
{
    "type": "CATALOG:ADD_LAYERS_FROM_CATALOGS",
    "layers": ["My GeoJSON Layer"],
    "sources": [{ "type":"GEOJSON", "url":"https://example.com/example.geojson" }]
}
```

`#/viewer/config?actions=[{"type":"CATALOG:ADD_LAYERS_FROM_CATALOGS","layers":["My GeoJSON Layer"],"sources":[{"type":"GEOJSON","url":"https://example.com/example.geojson"}]}]`

This GeoJSON catalog will return a single record similar to the 3D Tiles catalog and for this reason the name used in the layers array will be used to apply the title to the added vector layer.